modules/release-notes/partials/docs-server-7.6.6-release-note.adoc

```diff
@@ -1,13 +1,23 @@
 == Release 7.6.6 (May 2025)
 
-Couchbase Server 7.6.6 was released in May 2025.
-This maintenance release contains fixes several for known issues.
+[#deprecated-features-and-platforms-720]
+=== Deprecated and Removed Features and Platforms
+
+* The following operating systems are no longer supported:
```

Comment:
We're only deprecating in 7.2/7.6 for now (though we may remove them in a later release due to build/packaging issues if we need to), so these are still supported at this time.

Reply:
Okay, then I'll move this to the `7.2` section.